shared-libs/cht-script-api/src/auth.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * CHT Script API - Auth module
+ * Provides tools related to Authentication.
+ */
+
+const ADMIN_ROLE = '_admin';
```

Comment:
@dianabarsan should we check for `admin` as well? 
The admin document has the role without underscore:
<img width="403" alt="Screen Shot 2021-06-17 at 1 35 13 pm" src="https://user-images.githubusercontent.com/66472237/122344525-17a6e600-cf71-11eb-8041-2437f47f8fd8.png">
While the cookie has both: `{"name":"admin","roles":["_admin","admin"],"home":"/admin/"}`

Reply:
Thanks for the info! Instead of using the user-settings directly, I switched it to use the userCtx from session service 👍